client/src/components/Navbar.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link, useLocation } from 'react-router-dom'
+import styled from 'styled-components'
+
+const NavbarContainer = styled.div`
+    background-color: #B7C2B4;
+    display: flex;
+`
+
+const StyledLink = styled(Link)`
+    padding: 1em;
+    font-weight: bold;
+    color: black;
+    text-decoration: none;
+    font-size: 1.5em;
+    background-color: ${p => p.selected && '#A6B1A3'};
```

Comment:
I'm thinking we should use css variables for this (https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
You mean to abstract out the hex color code for reusability? styled-components offers the ThemeProvider component, which we can use for storing common colors/styles. I can make that in a future PR

For ref: [https://styled-components.com/docs/advanced#theming](url)